app/db/crud.py

```diff
@@ -0,0 +1,81 @@
+from uuid import UUID
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+from fastapi import HTTPException
+from app.db.models.builder import Keycap, Switch, Lubricant, Kits, Builds
+import typing
+
+
+def get_keycaps(db: Session) -> list[typing.Any]:  # type: ignore
+    return db.query(Keycap).all()
+
+
+def get_switches(db: Session) -> list[typing.Any]:  # type: ignore
+    return db.query(Switch).all()
+
+
+def get_lubricants(db: Session) -> list[typing.Any]:  # type: ignore
+    return db.query(Lubricant).all()
+
+
+def get_kits(db: Session) -> list[typing.Any]:  # type: ignore
+    return db.query(Kits).all()
+
+
+def get_builds(db: Session) -> list[typing.Any]:  # type: ignore
+    return db.query(Builds).all()
+
+
+# To do: Check whether the function has the correct logic.
+
+def keycap_info(db: Session, uuid: UUID):  # type: ignore
```

Comment:
Return type annotation. Same as above.

Rename to get_keycap instead. Same for others. keycap_info could mean updating keycap data